nevergrad/functions/__init__.py

```diff
@@ -3,6 +3,9 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+from .corefuncs import discrete_onemax as discrete_onemax
+from .corefuncs import discrete_jump as discrete_jump
+from .corefuncs import discrete_leadingones as discrete_leadingones
```

Comment:
It's simpler to import corefuncs in experiments.py instead

Reply:
Done.